infrastructure/crypto/README.md

```diff
@@ -0,0 +1,3 @@
+# The Crypto crate
+
+This crate is a standalone crypto library for use in Tari.
```

Comment:
Add major dependency information
Add install/build instructions as applicable

Reply:
Dependencies are easily found in `Cargo.toml` and listed if/when published to crates.io, so no need to repeat that here (and have it get out of date)

Reply:
install/build instructions are a good idea

Reply:
Dependencies: Agreed, no action

Reply:
Doh. This is a library, so build/install instructions aren't applicable.

As we add functionality, more examples will be good though.